src/test-clang.bats

```diff
@@ -14,7 +14,11 @@ clean() {
 
 testHelloCLLD() {
   clean
-  add clang lld
+  if [ "$(xx-info arch)" = "loong64" ]; then
```

Comment:
Why is this configured if all the tests are disabled(afaics).

I don't understand what installing binutils could do in here as binutils is for the native architecture only if not installed via xxadd.